### DISCUSSION TITLE
Video: Wrapping flatpickr with Alpine.js and Laravel Livewire (using @entangle)

### DISCUSSION THREAD

**Discussion by jonneroelofs** [livewire/livewire#4315] [2. Show and tell]
I made a video to show the general process of wrapping a vanilla js library with Alpine and Livewire. I use flatpickr as an example in the video.

[Wrapping flatpickr with Alpine.js and Laravel Livewire (using @entangle)](https://youtu.be/cLx40YxjXiw)

**Comment by blockhunts** [comment]
I followed your tutorial, and when I open my console. I got this error
Uncaught TypeError: Cannot convert undefined or null to object

What is the reason?
